Expensify/App
# Marker not updated properly fixed

@timszot 
### Details
The issue is specifically occurs in Mobile devices & mWeb

### Fixed Issues

$ #4357 

### Tests & QA Steps
1. Open new Expensify app
2. Wait for any new message on the chat list page
3. When a new message is received open the chat
4. Should show `NEW` indicator above the new messages
5. Go back to the chat list
6. Open the same chat again, `NEW` indicator should not be shown.
7.  Now go back to the chat list, & wait until a new message from the same user
8. Open the same chat now
9. Should show `NEW` indicator above the new messages
10. Go back to chat list
11. Open the same chat again, `NEW` indicator should not be shown.

### Tested On

### Screenshots


#### Mobile Web
https://user-images.githubusercontent.com/85645967/132128854-5560109e-b8f8-4bff-a151-426969fc1163.mp4

#### iOS
https://user-images.githubusercontent.com/85645967/132128985-71615ca1-0ebf-46a0-806f-35048d3c53e3.mov

#### Android
https://user-images.githubusercontent.com/85645967/132129234-ddf16cb1-58d1-4f06-b343-8b4acce4fd5f.mov
